### PR TITLE
Consume KUBEADMIN_PASSWORD and KUBECONFIG environment variables

### PR DESCRIPTION
**Pull request by OdedViner**
Added option to remove `cluster-path` parameter [from `run-ci` entrypoint], if there is an environment var named KUBECONFIG

#7498

### CODE REVIEW

**Comment by rabin-io** [comment]
The logic seems to be OK, we do assume both files will be on the same path, but that is reasonable.
My original thought was just setting the values from the ENV.

```python
            ocsci_config.RUN["kubeconfig_location"] = os.environ["KUBECONFIG"]
            ocsci_config.RUN["password_location"] = os.environ["KUBEADMIN_PASSWORD_FILE"]
```

File: ocs_ci/framework/pytest_customization/ocscilib.py [hunk 1]
```diff
@@ -497,7 +497,12 @@ def process_cluster_cli_params(config):
     suffix = ocsci_config.cur_index + 1 if ocsci_config.multicluster else ""
     cluster_path = get_cli_param(config, f"cluster_path{suffix}")
     if not cluster_path:
-        raise ClusterPathNotProvidedError()
+        if "KUBECONFIG" in os.environ:
```
Comment (dkeler) [inline ocs_ci/framework/pytest_customization/ocscilib.py:500]: `if os.getenv("KUBECONFIG"):`

File: ocs_ci/framework/pytest_customization/ocscilib.py [hunk 2]
```diff
@@ -497,7 +497,12 @@ def process_cluster_cli_params(config):
...
+            cluster_path = os.path.dirname(os.environ["KUBECONFIG"])
```
Comment (dkeler) [inline ocs_ci/framework/pytest_customization/ocscilib.py:501]: Do we need to check if `os.path.exists()`?

**Comment by OdedViner** [comment]
> The logic seems to be OK, we do assume both files will be on the same path, but that is reasonable. My original thought was just setting the values from the ENV.
> 
> ```python
>             ocsci_config.RUN["kubeconfig_location"] = os.environ["KUBECONFIG"]
>             ocsci_config.RUN["password_location"] = os.environ["KUBEADMIN_PASSWORD_FILE"]
> ```

image
the password location and kubeconfig location are on the same path.. We need to change it in many places..

File: ocs_ci/framework/pytest_customization/ocscilib.py (hunk 2 above)
Comment (OdedViner) [reply ocs_ci/framework/pytest_customization/ocscilib.py:501]:
```oviner:~$ printenv | grep -i kube
KUBECONFIG=/home/oviner/ClusterPath/auth/kubeconfig
```
We are checking the value of `KUBECONFIG`

File: ocs_ci/framework/pytest_customization/ocscilib.py (hunk 1 above)
Comment (OdedViner) [reply ocs_ci/framework/pytest_customization/ocscilib.py]: done

**Comment by rabin-io** [comment]
Also, you can't assume the filenames, as in the CI the files can be like `kubeadmin_<something>`, this why you need to use the reference from the environment variable as is.

**Review by petr-balogh — APPROVED** [review]

**Pull request closed without merging**